beesdoo_product_label/data/product_label.xml

```diff
@@ -61,9 +61,5 @@
             <field name="type">delivery</field>
             <field name="color_code">#ff4000</field>
         </record>
-        <record id="consignes_group_tax" model="account.tax.group">
-            <field name="name">Consignes</field>
-            <field name="sequence" eval="10" />
-        </record>
```

Comment:
this requires a version change on `beesdoo_product_label`, but since the version of `sale_product_deposit` is already updated in the branch, it will have to be merged with `nobump`. thus, the version of `beesdoo_product_label` should also be updated manually in this pull request.

in theory, it would require a major version change (since it breaks backwards compatibility), but since it is a quite new module (on its own), and this change is really a bug fix since this record should not be there, i would choose a patch release.

Reply:
done